devicon.json

```
@@ -3297,6 +3297,35 @@
         "color": "#5E5185",
         "aliases": []
     },
+    
+
+    {
+       "name": "hadoop",
+       "altnames": ["Apache Hadoop"],
+       "tags": [
+	  "framework",
+	  "big data"
+  	],
+  	"versions": {
+	    "svg": [ 
+	      "original",
+	      "plain"
+	    ],
+	    "font": [ 
+	      "original",
+	      "plain"
+	    ]
+	  },
+  	"color": "#fdeb4f", 
+  	"aliases": [
+  	   {
+  	   	"base": "plain",
+        "alias": "line"
+  	   }	
+  	] 
```

A few things need to be fixed:
- Indentation
- Remove `original` from `font`
- Change main color to the correct color: `#ffff00` (found in the official logo here: https://www.apache.org/logos/#hadoop)

(Note that the fill color also needs to be changed from `#feeb50` to `#ffff00` in [hadoop-original.svg](https://github.com/devicons/devicon/pull/1438/files#diff-6b3aedfa818bbe99cb6ac25a2d782b005498a9fe5ac775a517568641fe0942b6)

If you want, you can just apply the suggestion here (and change the [hadoop-original.svg](https://github.com/devicons/devicon/pull/1438/files#diff-6b3aedfa818bbe99cb6ac25a2d782b005498a9fe5ac775a517568641fe0942b6) file) 👇 
```suggestion
       "tags": [
      "framework",
      "big data"
      ],
      "versions": {
        "svg": [ 
          "original",
          "plain"
        ],
        "font": [ 
          "plain"
        ]
      },
      "color": "#ffff00", 
      "aliases": [
        {
          "base": "plain",
          "alias": "line"
        }	
      ] 
```